modernomad/settings/production.py

```diff
@@ -0,0 +1,29 @@
+from .common import *　# noqa
```

Comment:
```suggestion
from .common import * # noqa
```

Some weird special character here:

```
  File "/app/modernomad/settings/production.py", line 1
    from .common import *　# noqa
                         ^
SyntaxError: invalid character in identifier
```